source/gui/settingsDialogs.py

```diff
@@ -2546,6 +2546,27 @@ def onSave(self):
 		config.conf["documentFormatting"]["reportFrames"]=self.framesCheckBox.Value
 		config.conf["documentFormatting"]["reportClickable"]=self.clickableCheckBox.Value
 
+
+class DocumentNavigationPanel(SettingsPanel):
+	# Translators: This is the label for the document navigation settings panel.
+	title = _("Document Navigation")
+	helpId = "DocumentNavigation"

```

Comment:
This is expected to link to a section in the user guide

Reply:
https://github.com/nvaccess/nvda/blob/master/devDocs/userGuideStandards.md#feature-settings